md/single-sign-on-with-oidc.md

```diff
@@ -0,0 +1,316 @@
+
+::: warning  
+ Bonita "username" should match the prefered_username or one attribute of the subject returned by the IdP in the response. 
+ If some users need to be able to log in without having an account on the IdP, you can authorize it by activating an option in the file `authenticationManager-config.properties` (see 2. below). Users will then be able to log in using the portal login page (/login.jsp) provided they have a bonita account and their password is different from their username.  
+ You can configure Bonita engine to create the accounts on the fly in the database once a user accessing Bonita has been authenticated with the IdP (see the configuration of `bonita-tenant-sp-custom.properties` in the 3rd section of the next chapter).
+:::
+
+## Configure Bonita Bundle for OIDC
+
+You need to execute the following actions in the folder of each tenant for which you want to support authentication over OpenID Connect.
+If you want this configuration to also apply to each tenant created later, make sure to also perform those actions in the *template* tenant configuration folder:
+`<BUNDLE_HOME>/setup/platform_conf/current/tenant_template_*` (if you have not started the Bonita bundle yet, the files are located in `<BUNDLE_HOME>/setup/platform_conf/initial/tenant_template_*`)
+
+The bundle already contains the files needed to use OIDC with Bonita platform.
+To configure Bonita for OIDC:
+
+1. If you do not already have one:
+    1. Download a Subscription edition bundle from the customer portal
+    1. [Configure](_basic-bonita-platform-installation) it as needed
+    1. Run it a first time, so that the first default tenant is created (TENANT_ID = 1)
+    1. Stop it before modifying the configuration files below
+2. In the tenant_portal folder of each existing tenant: `<BUNDLE_HOME>/setup/platform_conf/current/tenants/<TENANT_ID>/tenant_portal`,
+   edit the authenticationManager-config.properties as follows:
+    ```
+       -->  #auth.AuthenticationManager = org.bonitasoft.console.common.server.auth.impl.standard.StandardAuthenticationManagerImpl
+            (...)
+       -->  auth.AuthenticationManager = org.bonitasoft.console.common.server.auth.impl.oidc.OIDCAuthenticationManagerImpl
+       -->  oidc.filter.active = true
+       -->  oidc.auth.standard.allowed = true
+       -->  oidc.logout.url = http://localhost:9090/auth/realms/bonita/protocol/openid-connect/logout
+       -->  oidc.logout.url.redirectQuery = ?redirect_uri={encodedRedirectUri}
+       -->  auth.tenant.admin.username = install
+       -->  auth.passphrase = BonitaBPM
+            (...)
+       -->  auth.tenant.standard.whitelist = william.jobs
+            (...)
+       -->  logout.link.hidden=true 
+    ```
+    
+    Make sure to [set the right tenant admin username](multi-tenancy-and-tenant-configuration#toc2).
```

Comment:
This will be removed in 7.13 (but it's not acted yet, so we can keep it)

Reply:
:+1: